src/main/java/org/zendesk/client/v2/model/UserProfile.java

```diff
@@ -0,0 +1,271 @@
+package org.zendesk.client.v2.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class UserProfile implements SearchResultEntity, Serializable {
+
+  private Map<String, Object> attributes;
+  private Date createdAt;
+  private String id;
+  private List<Identifier> identifiers;
+  private String name;
+  private String source;
+  private String type;
+  private Date updatedAt;
+  private String userId;
```

Comment:
looks odd to use a`String` here where the API (`public List<UserProfile> getUserProfilesForUser(long userId) {`) works with a `long`.